docs/bigcommerce-for-wordpress/extending-the-plugin/customization-guide.md

```diff
@@ -110,7 +121,7 @@ $height = $product->height;
 The product ID can appear in various places on the client side. The ID you use depends on context. Here are some places to look:

```

Comment:
The product ID can appear in various places on the client side. The ID you use depends on the context. Here are some places to look: